src/utils/comms.ts

```diff
@@ -21,7 +21,7 @@ export default {
         if (validRuntime && validEnv && validApi) {
             myParent.postMessage(msg, tgtOrigin, transfer);
         } else {
-            let errMsg = 'PureCloud Communication Failed.  ';
+            let errMsg = 'Genesys Cloud Communication Failed.  ';
```

Comment:
Let's switch this one back just on the off chance a client is using the string in an error check.  This could be thrown by any of the sdk methods.

And we'll get this one with the major release where where we switch the artifact from purecloud to genesysCloud.